includes/polling.php

```diff
@@ -403,33 +403,35 @@ function thold_check_all_thresholds() {
 
 	if (read_config_option('remote_storage_method') == 1) {
 		if ($config['poller_id'] == 1) {
-			$sql_query = "SELECT td.*, dtr.data_source_name
+			$sql_query = "SELECT td.*, dtr.data_source_name, h.hostname, h.description, h.notes, h.snmp_engine_id
 				FROM thold_data AS td
 				LEFT JOIN host AS h
 				ON h.id = td.host_id
 				LEFT JOIN data_template_rrd AS dtr
 				ON dtr.id = td.data_template_rrd_id
 				WHERE td.thold_enabled = 'on'
 				AND (h.poller_id = 1 OR h.poller_id IS NULL)
-				AND td.tcheck = 1";
+				AND td.tcheck = 1 AND h.status=3";
 		} else {
-			$sql_query = "SELECT td.*, dtr.data_source_name
+			$sql_query = "SELECT td.*, dtr.data_source_name, h.hostname, h.description, h.notes, h.snmp_engine_id
```

Comment:
put the h.hostname ... on a second line.  Same with the other examples.  Indent with hard tabs.